common/plugins/eu.esdihumboldt.hale.common.align/src/eu/esdihumboldt/hale/common/align/model/AlignmentUtil.java

```diff
@@ -1045,13 +1046,23 @@ public static void addValues(Group group, List<ChildContext> path,
 
 				// check all values
 				for (Object value : values) {
-					if (value instanceof Group) {
+					if (value instanceof InstanceDecorator) {
```

Comment:
Directly extracting the value here seems to ignore the `path` parameter that is passed in. It looks to me like that could lead to wrong results.